src/Adapter/Product/PackItemsManager.php

```diff
@@ -25,22 +25,24 @@
  */
 namespace PrestaShop\PrestaShop\Adapter\Product;
 
+use Pack;
+
 class PackItemsManager
 {
     /**
      * Get the Products contained in the given Pack.
      *
      * @param \Pack $pack
      * @param integer $id_lang Optional
-     * @return Array[Product] The products contained in this Pack, with special dynamic attributes [pack_quantity, id_pack_product_attribute]
+     * @return array(product) The products contained in this Pack, with special dynamic attributes [pack_quantity, id_pack_product_attribute]
```

Comment:
array of product i think, i just put array in lowercase because it wasn't valid (i did say it's valid now :smile: )